src/jobs/VebraAltoWrapperTask.php

```diff
@@ -119,6 +119,16 @@ public function execute($queue)
             foreach($fieldMapping as $craftField => $vebraField){
 
                 switch($vebraField){
+                    case 'housingType(category)':
+                        $ids = [];
+                        $testType = $property['type'][0];
```

Comment:
What if $property['type'] array is empty

Reply:
Added empty check and break if resultingly empty to next push